README.md

```diff
@@ -44,19 +44,37 @@ To build the operator:
     make build
     ```
 
-## Updating Custom Resource Definitions
-
-The custom resource definitions used by the Verrazzano Operator are auto generated by the Verrazzano project `verrazzano-crd-generator`.
-The steps to include changes made to the custom resource definitions are:
-
-* Make the API changes in the `verrazzano-crd-generator` project
-* Tag a branch of `verrazzano-crd-generator`, for example `0.n`
-* Edit `go.mod` to use the newly tagged API changes.  For example `github.com/verrazzano/verrazzano-crd-generator v1.2.0`
+## Updating Dependencies
+
+The dependencies used by the Verrazzano Operator are managed by go's `go.mod` file.
+Any license changes related to the new dependency versions should be carefully evaluated as discussed in the "Introducing a new dependency" section.  
+The steps to include changes made to dependencies are:
+
+* Run `go get` to update references to the dependency in the `go.mod` file.
```

Comment:
If you run go get does it automatically update the go.mod?  When i did this I explicitly updated go.mod then did a build, i did not run go get.

Reply:
Yes go get does update go.mod automatically.  Doing so has the benefit that it will translate the various inputs (i.e. hash, branch, tag) to a "canonical" version.  Of course go.mod can still be manually edited.